base/src/main/java/org/mozilla/jss/ssl/javax/JSSSession.java

```diff
@@ -315,4 +325,21 @@ public int getPeerPort() {
     public void setPeerPort(int port) {
         peerPort = port;
     }
+
+    public String getLocalAddr() {
+        return localAddr;
+    }
+
+    public void setLocalAddr(String localAddr) {
+        this.localAddr = localAddr;
+    }
+
+    public String getRemoteAddr() {
+        return remoteAddr;
+    }
+
+    public void setRemoteAddr(String remoteAddr) {
+        this.remoteAddr = remoteAddr;
+    }
```

Comment:
The class already has a `peerHost`. Should we just use that instead of adding a `remoteAddr`?

Reply:
> The class already has a `peerHost`. Should we just use that instead of adding a `remoteAddr`?

The peerHost contains the hostname of the client and is assigned by [JSSEngine](https://github.com/dogtagpki/jss/blob/60ef42cb76dc85d922a7677943f17b7afe302e6e/base/src/main/java/org/mozilla/jss/ssl/javax/JSSEngine.java#L241). The audit has the IP addresses of client and server.

Reply:
@ladycfu Do we need to capture IP addresses specifically, or are hostnames fine too?

Reply:
the current version of pp says we are allowed to provide at least one of  [IP Address, FQDN, user FQDN, or cert DN], however it also added that in the future, they will all be required.